docs/app/assets/css/prettify.css

```diff
@@ -1,3 +1,5 @@
+/* Modifiying a deep file */ 
+
 .pln { color: #000 }  /* plain text */
```

Comment:
**‘Add single comment’** – Leaving a comment on a file within the effective diff.

Reply:
Reply to conversation.